rules/gluster-utilization.libsonnet

```diff
@@ -0,0 +1,16 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'gluster_volume_capacity.rules',
```

Comment:
This should be part of `gluster_volume_utilization.rules`
https://github.com/gluster/gluster-mixins/pull/14/files#diff-87361da288c39093b2a433fd60b0b320R5

Reply:
I accept but I feel we should create different directories for rules, dashboards and alerts. Thats the reason I have done this way.

Reply:
We will have different directories for rules, dashboards and alerts. 
`name: 'gluster_volume_capacity.rules'` basically creates a new group of recording rules.
What I am suggesting is all the volume related rules should be under one group, that is `gluster_volume_utilization.rules` or we can have a more generic `gluster_volume.rules` group.

Reply:
I feel `gluster_volume.rules` makes sense and under this we can have all the recording rules related to volumes.

Reply:
made required changes.

Reply:
Ack. Will take care in this PR as well accordingly and update it.